"Tests / Windows (free-threading) / Build and test (x64, switch-case)" failed with:

```
FAIL: test_generator_not_under_consumer_arithmetic (test.test_profiling.test_sampling_profiler.test_blocking.TestBlockingModeStackAccuracy.test_generator_not_under_consumer_arithmetic)
Test that fibonacci_generator doesn't appear when consume_generator does arithmetic.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\cpython\cpython\Lib\test\test_profiling\test_sampling_profiler\test_blocking.py", line 137, in test_generator_not_under_consumer_arithmetic
    self.assertEqual(invalid_stacks, 0,
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
        f"Found {invalid_stacks}/{arithmetic_samples} invalid stacks where "
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        f"fibonacci_generator appears in the stack when consume_generator "
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        f"is on an arithmetic line. This indicates torn/inconsistent stack "
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        f"traces are being captured.")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 639 != 0 : Found 639/4508 invalid stacks where fibonacci_generator appears in the stack when consume_generator is on an arithmetic line. This indicates torn/inconsistent stack traces are being captured.


======================================================================
ERROR: test_getwche (test.test_msvcrt.TestConsoleIO.test_getwche)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\cpython\cpython\Lib\test\test_msvcrt.py", line 102, in test_getwche
    self.check_getwch('getwche')
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "D:\a\cpython\cpython\Lib\test\test_msvcrt.py", line 92, in check_getwch
    self.run_in_separated_process(code)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "D:\a\cpython\cpython\Lib\test\test_msvcrt.py", line 70, in run_in_separated_process
    subprocess.run(cmd, check=True, capture_output=True,
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                   creationflags=subprocess.CREATE_NEW_CONSOLE)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\cpython\cpython\Lib\subprocess.py", line 692, in run
    raise CalledProcessError(retcode, process.args,
                             output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['D:\\a\\cpython\\cpython\\PCbuild\\amd64t\\python_d.exe', '-c', '\nimport msvcrt\nfrom _testconsole import write_input\nwith open("CONIN$", "rb", buffering=0) as stdin:\n    write_input(stdin, b\'W[\')\n    assert msvcrt.getwche() == "å­—"\n']' returned non-zero exit status 3221225794.

======================================================================
FAIL: test_python_legacy_windows_stdio (test.test_cmd_line.CmdLineTest.test_python_legacy_windows_stdio)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\cpython\cpython\Lib\test\test_cmd_line.py", line 1047, in test_python_legacy_windows_stdio
    self.assertEqual(p.returncode, 0)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
AssertionError: 3221225794 != 0
```

3221225794 is 0xc0000142 (STATUS_DLL_INIT_FAILED) error code.

I restarted the job.

--- a/Lib/test/memory_watchdog.py
+++ b/Lib/test/memory_watchdog.py
@@ -19,6 +19,8 @@ def watchdog(pid):
             # fail the same way.
             return
 
+        # Prefer sys.stdout.write() to print() to use a single write() syscall.
+        # print(msg) calls write(msg.encode()) and then write(b"\n").
         sys.stdout.write(f" ... process data size: {mem / ONE_GIB:.1f} GiB\n")
         sys.stdout.flush()
         time.sleep(1)

--- a/Lib/test/memory_watchdog.py
+++ b/Lib/test/memory_watchdog.py
@@ -1,8 +1,5 @@
 """Memory watchdog: periodically read the memory usage of the main test process
 and print it out, until terminated."""
-# stdin should refer to the process' /proc/<PID>/statm: we don't pass the
-# process' PID to avoid a race condition in case of - unlikely - PID recycling.
-# If the process crashes, reading from the /proc entry will fail with ESRCH.
 
 
 import sys
@@ -28,7 +25,7 @@ def watchdog(pid):
 
 def main():
     if len(sys.argv) != 2:
-        print("usage: python {sys.argv[0]) pid")
+        print(f"usage: python {sys.argv[0]} pid")
         sys.exit(1)
     pid = int(sys.argv[1])
 

